route/dns.go

```diff
@@ -29,66 +27,82 @@ import (
 
 //TODO(auxten): this whole file need to be implemented
 
+type ResolveCache map[proto.RawNodeID]string
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported type ResolveCache should have comment or be unexported